examples.md

```diff
@@ -295,6 +296,24 @@ The yarn 2 cache directory will depend on your config. See https://yarnpkg.com/c
       ${{ runner.os }}-yarn-
 ```
 
+## Node - Jest
+The Jest cache directory will depend on your config. See https://jestjs.io/docs/cli#--cache for more info.
+
+```yaml
+- name: Get Jest cache directory path
+  id: jest-cache-dir-path
+  run: |
+    JEST_CACHE_DIR=$(yarn jest --showConfig | grep -o '"cacheDirectory": "[^"]*' | grep -o '[^"]*$')
```

Comment:
Will this work on Windows runners?
I'm not sure if `grep` will be available.

Reply:
@neilime Please see if you can create a separate example for windows (maybe using findstr), and then we can review this further. I want to avoid adding an incomplete example which will work on mac/linux but not on windows.

Reply:
IMO @vsvipul if we want to provide a platform agnostic solution, we should use github-script instead of shell.

As you can see in this example I've made: https://github.com/neilime/test-jest-cache/runs/7466470568?check_suite_focus=true

What do you think about it?

Reply:
This regex looks very complex for an example tbh. Can it be simplified somehow?